doc/build-hooks.rst

```diff
@@ -0,0 +1,60 @@
+Build Hooks
+===========
+
+Build hooks are programs that are run before (pre-build hook) and
+after (post-build hook) a package (including package dependencies)
+is built. The hooks are completely generic and can even be absent
+(their absence is ignored). Regardless of the return code of the
+pre-build hook, the normal build is executed. In the case where
+the pre-build hook provides a pre-built version of what the build
+step would provide, the build step is still run, but should be
+little more than a NOOP.
+
+Build hooks are project local rather than global to the user
+because a single user may want to use one set of hooks in one
+project and another set of hoos (or even none at all) for another
```

Comment:
TYPO: project and another set of `hoos` (or even none at all) for another

=> `hoos` needs a `k `

`project and another set of hooks (or even none at all) for another`